overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -47119,7 +47119,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "§bMultiblock Tweaker§r adds an §einteractive in-world preview§r for all §bGregTech§r multiblocks! §6Shift-right-click§r a placed controller to activate a hologram depicting the default layout of the structure to help you with building. As you place blocks, the structure will be validated and the §efirst incorrect block§r it detects will be highlighted in red.\n\n§6Sneak-right-click§r to cycle from the full view through each layer. When on the final layer of the preview, this will remove the preview.\n\n§6Sneak-left-click§r to go back to the previous layer, or close the preview from the full view.\n\nIn the §eJEI previews§r for multiblocks, §6left-click§r and drag the mouse to rotate the preview horizontally. §6Shift-left-click§r to rotate it vertically.\n\nFor §bMultiblock Tweaker§r structures, you can additionally §6right-click§r and drag the mouse to pan the preview or §6shift-right-click§r and drag to zoom in or out.",
+          "desc:8": "You can §6sneak-right click§r the controllers of §bGregTech§r multiblock machines with an empty hand to enable §ethe interactive in-world preview§r! This will show you a hologram of how a machine must be built. However, it has a small quirk of displaying §ethe first errored block only§r.\n\n§6Sneak-right click§r again to toggle the per-layer display.\n\n§6Sneak-left click§r to go back a layer.",
```

Comment:
Changes here

Reply:
Yeah I goofed. I need to revert the changes to quest text. I thought it was whitespace but I didn't look closely enough.